### PR TITLE
Removes Shoves Knockdowns Against Solid Objects

### PR DESCRIPTION
**Pull request by MrLostman**
## Changelog
:cl:
balance: removes shove knockdowns against solid objects
/:cl:

### CODE REVIEW

**Comment by MCterra10** [comment]
But why tho

**Comment by MCterra10** [comment]
How exactly is this bad gameplay, meme man

**Comment by Apple-107** [comment]
Do it.

File: tgstation.dme [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "austation\define_includes.dm"
```
Comment (MCterra10) [inline tgstation.dme]: Untick this file, loser.
It's already included further up

**Comment by MCterra10** [comment]
Tbh I like the gameplay of having to avoid walls and objects better than the gameplay of standing next to someone and spamming disarm on each other until you see who falls over first. Current shove gameplay actually requires effort to master and isn't entirely rng based. This meme is cringe

**Comment by AffectedArc07** [comment]
My real problem here is you used your master branch for this

File: tgstation.dme (hunk 1 above)
Comment (MrLostman) [reply tgstation.dme]: done

**Comment by MCterra10** [comment]
Lmao this is a salt PR you literally got dunked on by TLB after he shoved you ahahahah

**Pull request closed without merging**